bio/grit_genomics/wrapper.py

```
@@ -0,0 +1,38 @@
+extra = snakemake.params.get("extra", "")
+log = snakemake.log_fmt_shell(stdout=False, stderr=True)
+
+input_args = ""
+output_args = f" > {snakemake.output[0]}"
+if command in ("generate"):
+    output_args = f" --output {snakemake.output[0]}"
+    log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+elif command in ("sort", "merge", "slop", "complement", "genomecov", "multiinter"):
+    input_args = f" --input {snakemake.input.bed}"
+elif command in ("coverage", "intersect", "closest", "window", "jaccard", "subtract"):
+    input_args = f"--file-a {snakemake.input.bed[0]} --file-b {snakemake.input.bed[1]}"
```

Ruff (0.15.18)

[error] Undefined name `snakemake`

(F821)